src/Http/Middleware/ShareInertiaData.php

```diff
@@ -37,6 +37,11 @@ public function handle($request, $next)
                     'hasTermsAndPrivacyPolicyFeature' => Jetstream::hasTermsAndPrivacyPolicyFeature(),
                     'locale' => app()->getLocale(),
                     'managesProfilePhotos' => Jetstream::managesProfilePhotos(),
+                    'canUpdateProfileInformation' => Features::enabled(Features::updateProfileInformation()),
+                    'canUpdatePassword' => Features::enabled(Features::updatePasswords()),
```

Comment:
The `canUpdatePassword` prop is already set above, same goes for `canUpdateProfileInformation`, `managesProfilePhotos`, `canUpdateProfileInformation`. Is this on purpose? Implementations are almost the same